packages/chip-input/src/ux-choice-attribute.ts

```diff
@@ -0,0 +1,45 @@
+import { UxChoiceContainerAttribute } from './ux-choice-container-attribute';
+import { customAttribute, inject, observable, Optional } from 'aurelia-framework';
+import './ux-choice.css';
+
+@customAttribute('ux-choice')
+@inject(Element, Optional.of(UxChoiceContainerAttribute))
+export class UxChoiceAttribute {
+
+  public value: any;
+  @observable public selected: boolean;
+
+  constructor(private element: Element, private container: UxChoiceContainerAttribute) {}
+
+  public bind() {
+    this.container.registerChoice(this);
+  }
+
+  public attached() {
+    this.element.classList.add('ux-choice');
```

Comment:
do we need to wait until `attached` to add the class? If not, adding it early before `attached` could have perf improvement, since browser doesn't need to recalculate again due to style changes